GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs

```diff
@@ -44,7 +44,7 @@ public abstract class GitRepoTests
             "foo.cpp", // Changes to a folder in one test
             "FilenameEncoding",
             "GitCommandsTests",
-            "GVFLT_MultiThreadTest",
+            "GVFLT_MultiThreadTest", // Changes case in one test

```

Comment:
> Changes case in one test

I'm not sure what this comment means, could you add a few more details to the comment?

Reply:
Sure, will do when I get back from an offsite meeting -- stay tuned, and thank you again!

Reply:
I hope I've clarified that one comment sufficiently!

Reply:
Thanks looks good!

Reply:
OK -- will merge shortly!  Thank you!